CHANGELOG.md

```diff
@@ -7,178 +7,185 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.2.36] - 2024-09-24
```

Comment:
remove this line, you added unreleased features. The release process will inject this line when it actually bumps to 0.2.36